http://gcc.gnu.org/ml/gcc-patches/2006-03/msg01152.html

http://bugs.gentoo.org/145028

2006-05-04  Joseph S. Myers  <joseph@codesourcery.com>

	* config/mips/linux64.h (DWARF_OFFSET_SIZE): Remove.

Index: trunk/gcc/config/mips/linux64.h
===================================================================
--- trunk/gcc/config/mips/linux64.h	(revision 113523)
+++ trunk/gcc/config/mips/linux64.h	(revision 113524)
@@ -65,11 +65,6 @@ Boston, MA 02110-1301, USA.  */
 #undef LOCAL_LABEL_PREFIX
 #define LOCAL_LABEL_PREFIX (TARGET_OLDABI ? "$" : ".")
 
-/* The size in bytes of a DWARF field indicating an offset or length
-   relative to a debug info section, specified to be 4 bytes in the DWARF-2
-   specification.  The SGI/MIPS ABI defines it to be the same as PTR_SIZE.  */
-#define DWARF_OFFSET_SIZE PTR_SIZE
-
 /* GNU/Linux doesn't use the same floating-point format that IRIX uses
    for long double.  There's no need to override this here, since
    ieee_quad_format is the default, but let's put this here to make
